src/ui/Analytics/Analytics.ts

```diff
@@ -463,6 +492,31 @@ export class Analytics extends Component {
     );
   }
 
+  private isGtmScriptPresent() {
```

Comment:
I added this so that even if we do end up setting `pushToGtmDataLayer` to `true` by default (which is currently the case), `handleAnalyticsEventReady` will typically only be called if a GTM container snippet is present in the page.

I made the test none too restrictive, as a GTM snippet can take various forms such as:

- https://stackoverflow.com/questions/47457797/rename-the-datalayer-parameter-for-the-new-gtag-js
- https://developers.google.com/tag-manager/quickstart
or